kover-gradle-plugin/docs/index.md

```diff
@@ -85,223 +757,374 @@ buildscript {
 apply plugin: 'org.jetbrains.kotlinx.kover'    
 ```
 
-## Using Kover tasks
-Using JVM projects and Android projects differs, use the recommendations below according to your project type.
+#### Application in submodule with Plugins DSL
 
-### Kotlin JVM project
-These are projects that use Kotlin Gradle plugin.
+If you are using a plugin in a multi-module build, then there is no need to re-specify the plugin version in the submodules, 
+and it is enough to write like this:
+```kotlin
+plugins {
+     id("org.jetbrains.kotlinx.kover")
+}
+```
 
-For such projects, only the [Kover default tasks](#kover-default-tasks) for generating reports are used: `koverHtmlReport`, `koverXmlReport`, `koverVerify`.
+### Specify verification rules for total variant
```

Comment:
It seems that you have `Specify verification rules for total variant` and `Specify verification rules for named variant` sections before the actual documentation for verification rules